cross/monitoring-plugins/PLIST

```diff
@@ -0,0 +1,51 @@
+bin:bin/check_apt
+bin:bin/check_breeze
+bin:bin/check_clamd
+bin:bin/check_cluster
+bin:bin/check_dhcp
+bin:bin/check_disk
+bin:bin/check_disk_smb
+bin:bin/check_dummy
+bin:bin/check_file_age
+bin:bin/check_flexlm
+bin:bin/check_ftp
+bin:bin/check_http
+bin:bin/check_icmp
+bin:bin/check_ide_smart
+bin:bin/check_ifoperstatus
+bin:bin/check_ifstatus
+bin:bin/check_imap
+bin:bin/check_ircd
+bin:bin/check_load
+bin:bin/check_log
+bin:bin/check_mailq
+bin:bin/check_mrtg
+bin:bin/check_mrtgtraf
+bin:bin/check_nagios
+bin:bin/check_nntp
+bin:bin/check_nt
+bin:bin/check_ntp
+bin:bin/check_ntp_peer
+bin:bin/check_ntp_time
+bin:bin/check_nwstat
+bin:bin/check_oracle
+bin:bin/check_overcr
+bin:bin/check_ping
+bin:bin/check_pop
+bin:bin/check_procs
+bin:bin/check_real
+bin:bin/check_rpc
+bin:bin/check_sensors
+bin:bin/check_smtp
+bin:bin/check_ssh
+bin:bin/check_swap
+bin:bin/check_tcp
+bin:bin/check_time
+bin:bin/check_udp
+bin:bin/check_ups
+bin:bin/check_users
+bin:bin/check_wave
+bin:bin/negate
+bin:bin/urlize
+bin:bin/utils.pm
```

Comment:
Only ELF-binary files must be declared as `bin`. So .pm and .sh are only `rsc`.
Refer to https://github.com/SynoCommunity/spksrc/wiki/PLIST-files